contrib/completion/bash/docker

```diff
@@ -58,32 +59,56 @@ __docker_q() {
 	docker ${host:+-H "$host"} ${config:+--config "$config"} 2>/dev/null "$@"
 }
 
-__docker_complete_containers_all() {
-	local IFS=$'\n'
-	local containers=( $(__docker_q ps -aq --no-trunc) )
-	if [ "$1" ]; then
-		containers=( $(__docker_q inspect --format "{{if $1}}{{.Id}}{{end}}" "${containers[@]}") )
+# Returns a list of containers. Additional arguments to `docker ps`
```

Comment:
This would be the first and only comments that follow this style.
For consistency, I recommend we keep it as-is.